src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -207,10 +208,7 @@ public async Task Initialize(int version, bool runAllInitialization, Cancellatio
             await InitializeBase(cancellationToken);
 
             // If we are applying a full snap shot schema file, or if the server is just starting up
-            if (runAllInitialization)
```

Comment:
nit: should we remove this flag from method signature too if this is not needed anymore in the body?

Reply:
You are 100% right! Forgot to do this. Thank you!

Reply:
fixed!